Add new sdkroot_env_* output

jaimergp opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->


Closes #5 

cc @isuruf (is this what you had in mind?)

jaimergp committed:
Add new sdkroot_env_* output

jaimergp committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.11.12.35.56

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.9

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-recipe-manager`. The recipe can only be automatically migrated to the new v1 format if it is parseable by conda-recipe-manager.
 * ℹ️ sdkroot_env_cross_target_platform output has some malformed specs:
- In section test: `macosx_deployment_target_cross_target_platform _MACOSX_DEPLOYMENT_TARGET_ *_2`
Requirement spec fields should match the syntax `name [version [build]]`to avoid known issues in conda-build. For example, instead of `name =version=build`, use `name version.* build`. There should be no spaces between version operators and versions either: `python >= 3.8` should be `python >=3.8`.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19660243895](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19660243895). Examine the logs at this URL for more detail.</sub>

jaimergp committed:
Fix filenames

isuruf committed:
Modify activate-sdkroot_env.sh for cross-compilation

Updated SDKROOT environment activation script to handle cross-compilation on non-macOS platforms.

beckermr reviewed (commented):
I don't see how this pr follows the original issue. If sdkroot is already set, the script just exits. It doesn't always use the conda build sysroot.

jaimergp committed:
fix filename to copy too

jaimergp commented:
> I don't see how this pr follows the original issue. If sdkroot is already set, the script just exits. It doesn't always use the conda build sysroot.

I wasn't sure if we wanted to force ours or respect a previously set value by the user. Looks like we actually need it when cross-compiling too (because we can't use the conda sysroot?). Happy to adjust if we agree we prefer force-setting it.

jaimergp commented on recipe/activate-sdkroot_env.sh:
```diff
@@ -0,0 +1,17 @@
+    exit 0
+fi
+
+if [ "$(uname)" != "Darwin" ]; then
+    echo "Need to set SDKROOT when cross-compiling" >&2
+    exit 1
+fi
```
Since we are building on Linux runners, this makes the tests fail @isuruf. Shall we change to macOS runners or adjust this a bit so it doesn't fail on its own CI? (e.g. exclude for `PKG_NAME==sdkroot_env_*`).

isuruf replied:
Yes, let's change to macOS runners.

isuruf committed:
use return instead of exit

isuruf committed:
fix tests

isuruf commented:
@conda-forge-admin, rerender

Commit:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.11.23.16.08.1

isuruf commented:
Oops. I forgot I'm not a maintainer here.
